index.html

```diff
@@ -1,17 +1,40 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset='UTF-8'>
-    <title>Saimniecīsko darbu uzskaites sistēma</title>
 
-    <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
+<head>
+  <meta charset='UTF-8'>
+  <title>Saimniecisko darbu uzskaites sistēma</title>
 
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.css">
-    <link rel='stylesheet' href='./static/css/main.css'>
-  </head>
-  <body>
-    <div class="col-md-12">
-        <h1>Saimniecīsko darbu uzskaites sistēma</h1>
+  <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;700&display=swap" rel="stylesheet">
+  <link rel='stylesheet' href='./static/css/main.css'>
+  <link rel='stylesheet' href='./static/css/index.css'>
+</head>
+
+<body>
+  <div class="col-md-12">
+    <div class="nosaukums">
```

Comment:
Vai mums vajag šo un pārējās klases? Ar bootstrap kaut kas nesanāca?

Būtu forši, ja klases tad ieturētu vienā stilā, proti, angliski.